server/tests/auth.test.js

```diff
@@ -0,0 +1,62 @@
+import request from 'supertest-as-promised';
+import httpStatus from 'http-status';
+import jwt from 'jsonwebtoken';
+import chai, { expect } from 'chai';
+import app from '../../index';
+
+const config = require('../../config/env');
+const should = require('chai').should();
```

Comment:
@osahner Its recommended to be consistent in the assertions being used. Since we are using `expect` assertions so let's just continue using the same.